Makefile

```diff
@@ -76,8 +76,8 @@ endif
 
 depends:
 	apt-get update
-	apt-get install -y llvm-6.0 clang-6.0 libclang-6.0-dev \
-		linux-headers-4.4.0-98-generic linux-headers-4.10.0-14-generic \
+	apt-get install -y llvm-8 clang-8 libclang-8-dev \
+		linux-headers-4.11.0-14-generic linux-headers-4.10.0-14-generic \
```

Comment:
why not `linux-headers-${KERNEL_HEADER_VERSION}`? Do we need both the aarch64 and x64_64 header versions on every install? I would assume we only want the one we are currently building for. 

Similarly, maybe not super important but if the llvm and clang versions are so co-dependent maybe we should have a variable for the version and do something like:

```bash

CLANG_VERSION=8

CC = clang-${CLANG_VERSION}
LLC = llc-${CLANG_VERSION}
OPT = opt-${CLANG_VERSION}
LLVM_DIS = llvm-dis-${CLANG_VERSION}

apt-get install -y llvm-${CLANG_VERSION} clang-${CLANG_VERSION} libclang-${CLANG_VERSION}-dev \
        linux-headers-${KERNEL_HEADER_VERSION}

```

Although that may look too messy unsure.

Reply:
This looks cleaner than what we currently have.

Reply:
Yea I think that should work fine